x-pack/plugins/siem/public/components/field_renderers/field_renderers.test.tsx

```diff
@@ -211,10 +213,35 @@ describe('Field Renderers', () => {
   describe('#reputationRenderer', () => {
     test('it renders correctly against snapshot', () => {
       const wrapper = shallowWithIntl(
-        <TestProviders>{whoisRenderer('10.10.10.10')}</TestProviders>
+        <TestProviders>{reputationRenderer('10.10.10.10')}</TestProviders>
```

Comment:
NOTE: This looked like it was calling the wrong function by accident we overlooked, so I fixed the test here.